sentry_sdk/integrations/clickhouse_driver.py

```diff
@@ -118,9 +118,24 @@ def _inner_end(*args: P.args, **kwargs: P.kwargs) -> T:
                 span.set_data("db.result", res)
 
             with capture_internal_exceptions():
-                span.scope.add_breadcrumb(
-                    message=span._data.pop("query"), category="query", data=span._data
-                )
+                query = span._get_attribute("db.query.text")
```

Comment:
If we need to access attributes outside, let's officially expose `get_attributes` or a `@property` called `attributes`. Accessing underscore methods outside the implementation is an anti-pattern and we should stop doing that.